spec/models/case_cycle_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CaseCycle, type: :model do
+  let(:invalid_date_msg) { 'Invalid date' }
+
+  it { should validate_numericality_of(:copay).is_greater_than(0) }
+
```

Comment:
Need specs/examples/tests for `status`:

```
it {
    should define_enum_for(:status).with_values(
      submitted: 'submitted',
      pending: 'pending',
      approved: 'approved',
      denied: 'denied'
    ).backed_by_column_of_type(:enum) }

  it { should allow_values(:submitted, :pending, :approved, :denied).for(:status) }

```

Reply:
@weedySeaDragon Isn't the second test superfluous? 🤔

Reply:
Agreed.  Some folks like to have both:  they _specify_ slightly different things.  Even though I'm quite on the "these are _specifications!!_" side of things, there's not much real additional value to the 2nd; the values in an :enum also define the allowed ("enumerated") values. 
(I just copied and pasted from existing tests.  I should have actually reviewed what I pasted more carefully.)